api/src/services/listing-csv-export.service.ts

```diff
@@ -580,6 +580,30 @@ export class ListingCsvExporterService implements CsvExporterServiceInterface {
           path: 'servicesOffered',
           label: 'Services Offered',
         },
+        {
```

Comment:
issue: I just realized this wasn't added to the ticket, but the neighborhood amenities will only appear if a feature flag is turned on. Because of that we only want to show these columns in the export if the feature flag is turned on. 

You can reference the "homeType" column further up in this file. You'll also need to create the feature flag in the DB to see it so I'd recommend creating it in "seed-staging" like what we do for `enableUtilitiesIncluded`. The new feature flag should be called something like `enableNeighborhoodAmenities`

Reply:
@jaredcwhite, looks like you forgot to remove these after adding them behind the feature flag above